CircuitPy/code.py

```diff
@@ -8,14 +8,28 @@
 import busio
 import board
 
+
 import adafruit_vl53l0x
 
 # Initialize I2C bus and sensor.
-i2c1 = busio.I2C(board.GP5, board.GP4)
-vl53_1 = adafruit_vl53l0x.VL53L0X(i2c1)
+sensor1Range = -2.0
+sensor2Range = -2.0
+
+uart = busio.UART(board.GP0, board.GP1, baudrate=115200, bits=8, parity=None )
+    
+print("new version - uart")
+try:
+    i2c1 = busio.I2C(board.GP5, board.GP4)
+    vl53_1 = adafruit_vl53l0x.VL53L0X(i2c1)
+except:
+    print("No 1st sensor connected")
```

Comment:
Do these prints get saved anywhere where we could access them later? After a match for instance